### PR TITLE
Adds support for foreign key update and deletion actions

### PR DESCRIPTION
**Pull request by jlavelle**

### CODE REVIEW

**Comment by jlavelle** [comment]
@qxjit @telser FYI this is what the haddocks for `ForeignKeyOptions` look like without exporting the constructor: 
image

**Comment by jlavelle** [comment]
Changing the way the fields are exported at least does this: 
image

File: orville-postgresql-libpq/orville-postgresql-libpq.cabal [hunk 1]
```diff
@@ -1,6 +1,6 @@
 cabal-version: 1.12
 
--- This file has been generated from package.yaml by hpack version 0.34.4.
+-- This file has been generated from package.yaml by hpack version 0.35.0.
```
Comment (jlavelle) [inline orville-postgresql-libpq/orville-postgresql-libpq.cabal]: This was inadvertent and fixed with f4f09a4

**Review by qxjit — APPROVED** [review]
:heart: Looks good!

**Pull request merged**